packages/access/lib/expireGrants.js

```diff
@@ -0,0 +1,25 @@
+const debug = require('debug')('access:lib:expireGrants')
+
+const grantsLib = require('./grants')
+
+/**
+ * Renders expired grants invalid.
+ */
+module.exports = async ({ now, dryRun }, { t, pgdb, mail }) => {
```

Comment:
`now` and `dryRun` are unused and never filled. Did you do it to match the other schedulers? Might be a bit overkill here if totally unused.